src/background/services/background.ts

```diff
@@ -227,7 +233,6 @@ export class Background {
       const data = await this.storage.get()
       this.logger.info(data)
       if (details.reason === 'install') {
-        await this.storage.populate()
         await this.openPaymentsService.generateKeys()
```

Comment:
There's a chance `generateKeys` (which saves keys in storage) gets called before `Storage.populate()` still. I think duplicating `this.storage.populate()` here will be safe still.